.yamato/project-tests.yml

```diff
@@ -41,6 +41,9 @@ test_{{ project.name }}_{{ editor }}_{{ platform.name }}:
     flavor: {{ platform.flavor}}
   commands:
     - npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
+    - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+    - unity-downloader-cli -u {{ editor }} -c editor -w --fast
+    - {{ platform.editorpath }} -projectpath testproject -batchmode -quit -logfile BuildMultiprocessTestPlayer.log -executeMethod Unity.Netcode.MultiprocessRuntimeTests.BuildMultiprocessTestPlayer.BuildDebug 
```

Comment:
so, this is multiprocess testing in the same box/machine, right?

Reply:
I tried initially to do everything in one large PR earlier but it was too complicated and too many lines so I'm taking it one step at a time.

Reply:
no, I would actually prefer individual PRs going through :) I was just double-checking my understanding.